File: Java/Trailing-Zeros.java
class Solution {
    /*
     * param n: As desciption
     * return: An integer, denote the number of trailing zeros in n!
     */
    public long trailingZeros(long n) {
        // write your code here
        long num = 0;
        while (n != 0) {
            num += n / 5;
            n = n / 5;
        }
        return num;
    }
};

File: Cpp/Trailing-Zeros.cpp
class Solution {
public:
    // param n : description of n
    // return: description of return
    long long trailingZeros(long long n) {
        long long res = 5, cnt = 0;
        while (res <= n) {
            cnt += n / res;
            res *= 5;
        }
        return cnt;
    }
};
